cmd/tsbs_generate_queries/uses/devops/common.go

```diff
@@ -43,6 +43,18 @@ const (
 // for ease of testing
 var fatal = log.Fatalf
 
+// GetFatal allows us to get the current fatal logger function.
+// This is mostly to help with testing.
+func GetFatal() func(format string, v ...interface{}) {
```

Comment:
I understand why you needed this, but it seems a bit hacky to me, so I went and 'fixed' the underlying problem here:
https://github.com/timescale/tsbs/pull/77

It will also probably make the test methods above a bit more sane since you can just do a standard trick for capturing a panic

Reply:
I think I'll wait for that PR to be merged before changing this. Is that OK?